lib/src/compiler/expression_parser/ast.dart

```diff
@@ -74,6 +74,17 @@ class Conditional extends AST {
   }
 }
 
+class IfNull extends AST {
```

Comment:
Mind leaving a tiny bit of dartdoc here:

/// Represents the `??` expression in Dart